src/cmake/modules/FindOpenColorIO.cmake

```diff
@@ -24,13 +24,15 @@ include (FindPackageMessage)
             message(STATUS "OCIO LIBRARY_PATH explicitly specified: ${OCIO_LIBRARY_PATH}")
         endif()
     endif ()
-    FIND_PATH( OCIO_INCLUDES OpenColorIO/OpenColorIO.h
+    FIND_PATH(OCIO_INCLUDES
+        OpenColorIO.h
         ${OCIO_INCLUDE_PATH}
```

Comment:
Indeed the doc seems to be clear on that. I've overwrote the commit and added the token __PATHS__. It produces the correct result on my configuration (cmake 3.4.3).